test_apps/rails5.rb

```diff
@@ -9,7 +9,7 @@
 
 module RailsApp
   class Application < Rails::Application
-    config.root                                       = __dir__
+    config.root                                       = File.dirname(__FILE__)
```

Comment:
In that case, would it be ok if I remove ruby 1.9 from travis config ? 

I was just worried that a red cross besides my PR would hamper the changes of it being accepted :)